### ISSUE TITLE
New list of 23 auto-antibodies

### ISSUE THREAD

**Issue by MaudDLVT** [obophenotype/human-phenotype-ontology#6788]
**Preferred term label:**

**Synonyms**

**Definition (free text, please give PubMed ID)**

**Parent term (use hpo.jax.org/app)**

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**

**Your nano-attribution (ORCID)**

[20210309 Fimarad.xlsx](https://github.com/obophenotype/human-phenotype-ontology/files/6108546/20210309.Fimarad.xlsx)

**Comment by MaudDLVT** [comment]
Dear all,
Is there any news about my Excel file with the 23 new term requested?
Many thanks,
Maud

**Comment by pnrobinson** [comment]
Laminin 332 is the new name for laminin 5, I will merge these
see https://medlineplus.gov/genetics/gene/lama3/

**Comment by pnrobinson** [comment]
anti-transglutaminase 3 antibody is the same as -- 
Anti-epidermal transglutaminase antibody positivity HP:0033565
https://hpo.jax.org/app/browse/term/HP:0033565
Adding synonyms

**Comment by pnrobinson** [comment]
Anti-transglutaminase 2 antibody  is the same as
Anti-endomysial antibody positivity HP:0033637

I have added the synonym

**Comment by pnrobinson** [comment]
I think anti-endomysium antibody  is also the same as Anti-endomysial antibody positivity HP:0033637

**Comment by pnrobinson** [comment]
added all terms, thanks for the suggestions!
